.github/workflows/ci-test-local.yml

```diff
@@ -79,7 +79,8 @@ jobs:
         env:
           GOPATH: /home/runner/go
 
-      - uses: codecov/codecov-action@v3
+      - uses: actions/upload-artifact@v3
         if: ${{ always() }}
         with:
-          files: ./KubeArmor/gover.coverprofile
+          name: coverage
```

Comment:
This will end up in overwritting reports for each workflow. We should probably name it `coverage-${{ matrix.os }}-${{ matrix.runtime }}`